vignettes/adrs.Rmd

```diff
@@ -425,12 +447,53 @@ dataset_vignette(
 ## Derive Response Parameters requiring Confirmation {#confirm}
 
 Any of the above response parameters can be repeated for "confirmed" responses only. For these the functions
-`derive_param_confirmed_resp()` and `derive_param_confirmed_bor()` can be used. Here is an example of a BOR parameter
-when requiring confirmation. The assessment and the confirmatory assessment need to occur at least 28 days apart, using the
-`ref_confirm` argument.
+`derive_param_confirmed_resp()` and `derive_param_confirmed_bor()` can be used. Some of the other functions
+from above can then be re-used passing in these confirmed response records. See the examples below of derived parameters
+requiring confirmation. The assessment and the confirmatory assessment here need to occur at least 28 days apart,
+using the `ref_confirm` argument.
```

Comment:
Ad discussed , can we add when we say 28 days , its doesn't includes +1 in day calculation. This is same for ref_start_window and ref_confirm. This was mentioned in comments sheet.

Reply:
sure, will add this point to the vignette as a reminder. I saw that this comment also got taken care of in the function documentation pages by another issue/PR already.